tests/regression/test_model.py

```diff
@@ -1,5 +1,6 @@
 import numpy
 import pandas
+from pandas.testing import assert_frame_equal
```

Comment:
Exact testing of floats finally broke.  🤷 .  Should have been doing tolerance testing anyway.